packages/xdl/src/Project.js

```diff
@@ -1892,6 +1894,38 @@ async function stopWebpackServerAsync(projectRoot) {
   });
 }
 
+export async function bundleWebpackAsync(projectRoot, packagerOpts) {
+  let { exp } = await ProjectUtils.readConfigJsonAsync(projectRoot);
+  if (!exp.platforms.includes('web')) {
+    return;
+  }
+  let config = webpackConfig({
+    projectRoot,
+    development: packagerOpts.dev,
+    production: !packagerOpts.dev,
+  });
+  let compiler = webpack(config);
+
+  process.env.BABEL_ENV = 'production';
+  process.env.NODE_ENV = 'production';
+
+  await new Promise((resolve, reject) =>
+    compiler.run((error, stats) => {
+      // TODO: Bacon: account for CI
+      if (error) {
+        // TODO: Bacon: Clean up error messages
+        return reject(error);
+      }
+
+      const statsPath = projectRoot + '/web-build-stats.json';
+      return bfj
```

Comment:
We use `@expo/json-file` for basically all JSON reading/writing in this repo, so we could avoid the `bfj` dependency and use `@expo/json-file`:
```js
import JsonFile from `@expo/json-file`;
  // ...
  let stats = await new Promise((resolve, reject) =>
    compiler.run((error, stats) => {
      // TODO: Bacon: account for CI
      if (error) {
        // TODO: Bacon: Clean up error messages
        return reject(error);
      }
      resolve(stats);
    })
  );
  await JsonFile.write(path.join(projectRoot, 'web-build-stats.json'), stats.toJson());
```